### PR TITLE
update brainflow to 5.0.1

### PR DESCRIPTION
**Pull request by Andrey1994**
Signed-off-by: Andrey Parfenov <a1994ndrey@gmail.com>

* change bandstop from center and width to start and stop, first reason for it - it matches brainflow code, second - I think params in filters should be consistent between bandpass and bandstop
* removed classifiers from focus widget, I thought about removing dropdawn also but decided to keep it for now(we still can remove it)

### CODE REVIEW

File: OpenBCI_GUI/FocusEnums.pde [hunk 1]
```diff
@@ -92,10 +92,7 @@ public enum FocusMetric implements IndexingInterface
 
 public enum FocusClassifier implements IndexingInterface
 {
-    REGRESSION (0, "Regression", BrainFlowClassifiers.REGRESSION),
-    KNN (1, "KNN", BrainFlowClassifiers.KNN),
-    SVM (2, "SVM", BrainFlowClassifiers.SVM),
-    LDA (3, "LDA", BrainFlowClassifiers.LDA);
+    REGRESSION (0, "Regression", BrainFlowClassifiers.DEFAULT_CLASSIFIER);
```
Comment (retiutut) [inline OpenBCI_GUI/FocusEnums.pde:95]: @Andrey1994 What happened to the other classifiers?

Comment (Andrey1994) [reply OpenBCI_GUI/FocusEnums.pde:95]: https://brainflow.org/2022-06-09-100k-brainflow-5/

Comment (retiutut) [reply OpenBCI_GUI/FocusEnums.pde:95]: It's a little silly to have a dropdown now if there is only one option. 😕

File: OpenBCI_GUI/FocusEnums.pde [hunk 2]
```diff
@@ -47,8 +47,8 @@ public enum FocusXLim implements IndexingInterface
 
 public enum FocusMetric implements IndexingInterface
 {
-    CONCENTRATION (0, "Concentration", BrainFlowMetrics.CONCENTRATION, "Concentrating"),
-    RELAXATION (1, "Relaxation", BrainFlowMetrics.RELAXATION, "Relaxing");
+    CONCENTRATION (0, "Concentration", BrainFlowMetrics.MINDFULNESS, "Concentrating"),
+    RELAXATION (1, "Relaxation", BrainFlowMetrics.RESTFULNESS, "Relaxing");
```
Comment (retiutut) [inline OpenBCI_GUI/FocusEnums.pde:51]: Maybe we should update the text displayed in this dropdown. Concentration -> Mindfulness etc.

**Review by retiutut — COMMENTED** [review]
Tested and reviewed. Maybe we should make some small changes as mentioned. Core changes for this PR are good and functional.

File: OpenBCI_GUI/FocusEnums.pde (hunk 2 above)
Comment (Andrey1994) [reply OpenBCI_GUI/FocusEnums.pde:51]: I thought about it and decided to keep it as before to dont confuse anybody, its technically the same metric as before but with different name

**Pull request merged**